# RobotStatus.msg
sensor_msgs/JointState joint_state
float64[] joint_acceleration
float64[] force_torque
